modules/using-wildcard-routes.adoc

```diff
@@ -0,0 +1,317 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-controller
+
+[id="using-wildcard-routes_{context}"]
+= Using wildcard routes
+
+The HAProxy router has support for wildcard routes, which are enabled by setting
+the `ROUTER_ALLOW_WILDCARD_ROUTES` environment variable to `true`. Any routes
+with a wildcard policy of `Subdomain` that pass the router admission checks will
+be serviced by the HAProxy router. Then, the HAProxy router exposes the
+associated service, for the route, per the route's wildcard policy.
+
+[IMPORTANT]
+====
+To change a route's wildcard policy, you must remove the route and recreate it
+with the updated wildcard policy. Editing only the route's wildcard policy in a
+route's *.yaml* file does not work.
+====
+
+.Procedure
+
+----
+$ oc adm router --replicas=0 ...
+$ oc set env dc/router ROUTER_ALLOW_WILDCARD_ROUTES=true
+$ oc scale dc/router --replicas=1
```

Comment:
If these are steps to change a route's wildcardPolicy, the steps are oc delete route/foo and oc create -f $FOO_MANIFEST where $FOO_MANIFEST contains a wildcardPolicy: [Subdomain](https://github.com/openshift/api/blob/master/route/v1/types.go#L271)

Reply:
I added an updated procedure here, but can't test it because of my current cluster (I'm having trouble with the edit command, which I realize might not be correct).